src/main/java/io/cryostat/agent/insights/InsightsAgentHelper.java

```diff
@@ -67,15 +68,38 @@ public boolean isInsightsEnabled(PluginInfo pluginInfo) {
     }
 
     public void runInsightsAgent(PluginInfo pluginInfo) {
+        Map<String, String> out = new HashMap<>();
+
+        // Check if debug logging should be enabled
+        final String defaultLogLevel;
+        boolean debug =
+                config.getOptionalValue(RHT_INSIGHTS_JAVA_DEBUG, boolean.class).orElse(false);
+        if (debug) {
+            out.put("debug", "true");
+            defaultLogLevel = "debug";
+        } else {
+            // Otherwise, apply any log level set for the Cryostat agent
+            defaultLogLevel = System.getProperty(SHADE_PREFIX + SimpleLogger.DEFAULT_LOG_LEVEL_KEY);
```

Comment:
There is a class `io.cryostat.agent.shaded.ShadeLogger`, maybe rather than this `SHADE_PREFIX` constant this could use `ShadeLogger.class.getPackageName()`?

Reply:
Nice. I wasn't a fan of hard-coding that.